Source/FieldSolver/FiniteDifferenceSolver/MacroscopicProperties/MacroscopicProperties.cpp

```diff
@@ -161,30 +119,105 @@ MacroscopicProperties::InitData ()
     amrex::Print() << "we are in init data of macro \n";
     auto & warpx = WarpX::GetInstance();
 
-    IntVect Ex_stag = warpx.getEfield_fp(0,0).ixType().toIntVect();
-    IntVect Ey_stag = warpx.getEfield_fp(0,1).ixType().toIntVect();
-    IntVect Ez_stag = warpx.getEfield_fp(0,2).ixType().toIntVect();
-    IntVect Bx_stag = warpx.getBfield_fp(0,0).ixType().toIntVect();
-    IntVect By_stag = warpx.getBfield_fp(0,1).ixType().toIntVect();
-    IntVect Bz_stag = warpx.getBfield_fp(0,2).ixType().toIntVect();
+    // Get BoxArray and DistributionMap of warpx instant.
```

Comment:
Just a typo, I think:
```suggestion
    // Get BoxArray and DistributionMap of warpx instance.
```